src/snowflake/snowpark/stored_procedure.py

```diff
@@ -108,7 +108,10 @@ class StoredProcedureRegistration:
           function on the Snowflake server during stored procedure creation. During the serialization, the
           global variables used in the Python function will be serialized into the bytecode,
           but only the name of the module object or any objects from a module that are used in the
-          Python function will be serialized. During the deserialization, Python will look up the
+          Python function will be serialized. If the size of the serialized bytecode is over 8K bytes, it's uploaded to a stage location.
```

Comment:
```suggestion
          Python function will be serialized. If the size of the serialized bytecode is over 8K bytes, it will be uploaded to a stage location as a Python file.
```